cmd/geth/config.go

```diff
@@ -347,7 +347,7 @@ func setDefaultMumbaiGethConfig(ctx *cli.Context, config *gethConfig) {
 	config.Node.HTTPPort = 8545
 	config.Node.IPCPath = utils.MakeDataDir(ctx) + "/bor.ipc"
 	config.Node.HTTPModules = []string{"eth", "net", "web3", "txpool", "bor"}
-	config.Eth.SyncMode = downloader.SnapSync
+	config.Eth.SyncMode = downloader.FullSync
```

Comment:
When this flag was implemented it was FULL by default but then @jdkanani asked me to use SNAP and we changed it for both Mumbai and Mainnet.

Reply:
And I think the reason behind this was that Geth is using SNAP as their default sync mode.

Reply:
@jdkanani I got it correctly we're having issues with `snap` syncs right? Do we need to set this back to `full`?

Reply:
I was last working on the `snap` sync issue and it still remains unresolved for mainnet.

Reply:
Yes, we should change it to `full` until we get `snap` working on mainnet.